WireAPI/Tests/WireAPITests/Backend/Resources/certificates.json

```diff
@@ -1,7 +1,7 @@
 
 {
     "wire" : [
-        "MIIGvTCCBaWgAwIBAgIQBzPX66S/gK/nx9QwNwaKBjANBgkqhkiG9w0BAQsFADBZMQswCQYDVQQGEwJVUzEVMBMGA1UEChMMRGlnaUNlcnQgSW5jMTMwMQYDVQQDEypEaWdpQ2VydCBHbG9iYWwgRzIgVExTIFJTQSBTSEEyNTYgMjAyMCBDQTEwHhcNMjUwMzEyMDAwMDAwWhcNMjYwNDA5MjM1OTU5WjBKMQswCQYDVQQGEwJDSDEMMAoGA1UEBxMDWnVnMRgwFgYDVQQKEw9XaXJlIFN3aXNzIEdtYkgxEzARBgNVBAMMCioud2lyZS5jb20wggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEKAoIBAQCt5jMFa6+dUph+A01fd1WNSeohW2XhepCcJxjqb+xYzXlNMrRuj0UqczE0A+0PMHpWJG+lmwoR59fymLXklyzi5mK5nzUhJXVurG2myMnnpiN6Z730NxrlyTfmlOFi4rqNny8bqkmJj2ZFj2cZp2J3ipYvu7AB6gifHaY4zsd6kIKHY05d34SNDiwGx+Bv6RatxVCYHO8sc9QOjKSb+b4G8vZ4nWeM82Iz8ah5duYhbVYzeJ+5xgmgP2D5Xk18d8A2tW7bDhhwsNp3QLzk1vxTWyAU2SuA6rOF3/XEeiTW47KOh4tMgcdiSvK9sESZ2Xq/5/YnUQzT4WP2+x4jZNitAgMBAAGjggOOMIIDijAfBgNVHSMEGDAWgBR0hYDAZsffN97PvSk3qgMdvu3NFzAdBgNVHQ4EFgQU/0iA8JzB4tDtwNB/3NyYfCtfTZwwHwYDVR0RBBgwFoIKKi53aXJlLmNvbYIId2lyZS5jb20wPgYDVR0gBDcwNTAzBgZngQwBAgIwKTAnBggrBgEFBQcCARYbaHR0cDovL3d3dy5kaWdpY2VydC5jb20vQ1BTMA4GA1UdDwEB/wQEAwIFoDAdBgNVHSUEFjAUBggrBgEFBQcDAQYIKwYBBQUHAwIwgZ8GA1UdHwSBlzCBlDBIoEagRIZCaHR0cDovL2NybDMuZGlnaWNlcnQuY29tL0RpZ2lDZXJ0R2xvYmFsRzJUTFNSU0FTSEEyNTYyMDIwQ0ExLTEuY3JsMEigRqBEhkJodHRwOi8vY3JsNC5kaWdpY2VydC5jb20vRGlnaUNlcnRHbG9iYWxHMlRMU1JTQVNIQTI1NjIwMjBDQTEtMS5jcmwwgYcGCCsGAQUFBwEBBHsweTAkBggrBgEFBQcwAYYYaHR0cDovL29jc3AuZGlnaWNlcnQuY29tMFEGCCsGAQUFBzAChkVodHRwOi8vY2FjZXJ0cy5kaWdpY2VydC5jb20vRGlnaUNlcnRHbG9iYWxHMlRMU1JTQVNIQTI1NjIwMjBDQTEtMS5jcnQwDAYDVR0TAQH/BAIwADCCAXwGCisGAQQB1nkCBAIEggFsBIIBaAFmAHQAlpdkv1VYl633Q4doNwhCd+nwOtX2pPM2bkakPw/KqcYAAAGVihXnYAAABAMARTBDAiBz+VZgXR9DCBiIZzQVs88ERkxse3yX5dyBn6ZglCv7zgIfKQvPJh+bbOsi5W4PL42VgQA99ENB0NLdjLJ1Hy0kCwB3AGQRxGykEuyniRyiAi4AvKtPKAfUHjUnq+r+1QPJfc3wAAABlYoV52cAAAQDAEgwRgIhAIvVzPEpzKRVPyA/auaqhEsNQCdVwv7FoUHCfcYcD8AvAiEA+7hooWY4+nV1cEW+qBD0/fTrwQg/pEQe1i+AZQ2yhHAAdQBJnJtp3h187Pw23s2HZKa4W68Kh4AZ0VVS++nrKd34wwAAAZWKFed1AAAEAwBGMEQCIDEaxK7crVRRtFbfwRb2kAouKQoR3D61gP1I1q0ct2f6AiBKaR+KnZPL2JPCYw45E4VvGKq6ZKBzmZEDBWCPUyaWgDANBgkqhkiG9w0BAQsFAAOCAQEAxXgInZLCitZ5JZe+i/NIrXrv2cDLrO6sFmqP0sLiKztLUYRyTXNZFK71uuQ4cmYViPBVYvekcdm0FYew0C9wEbEsKVQ2+P4OZYmtARFUw0nqlH/hqoh/xR4TDycoaXL0Y9vcbv0x+gSZOJDYyLHv35f0f2HJ6KwyIgSAJmGazR37/euR28ZS7jGG3gZTMyzC8SRkhfFC4zaJF71gj8qsX50jN5cllBNMDmiYnJVp9QjyKXGpFu4/zS3pzN8JIgLgLb+fjIFfcREagFi4+OzJ0NNOti8MZrxyalRmpR8sVstywWSj6LmAfYQwwsX5ZCz45lLAy1xZVB0UtXtulYYb6g==",
```

Comment:
Restoring the certificate [PR](https://github.com/wireapp/wire-ios/pull/2868) updated as a quick fix.
This PR is supposed to be the correct long-term fix.